TestProject/src/main/java/Test.java

```diff
@@ -0,0 +1,10 @@
+public class Test {
```

Comment:
All classes should be inside a package (e.g. app), we should avoid using classes that are in the default package, even for tests